tier4_system_msgs/msg/CommandModeAvailabilityItem.msg

```diff
@@ -0,0 +1,2 @@
+string mode
```

Comment:
Do we want to define this as string?
Are there any reasons for not defining constant values for each mode?

Reply:
Because the mode includes system-dependent MRM, it could not be commonly defined. Therefore, it was created as a string that is easy to expand.

Feedback from evaluations has suggested that number would be better, but this would make the setting file very difficult to read, as shown below.
```diff
- command_modes: [manual, stop, autonomous, local, remote, pull_over, comfortable_stop, emergency_stop]
+ command_modes: [101, 201, 202, 203, 204, 303, 302, 301]

```

Since it is a trade-off with human readability, I'm considering which to use. However, since it is currently working with strings, and the impact of changing it to number will only affect to the internal interface. So I would like to adopt string at first.